### PR TITLE
Update Iris Rev 3 QMK-DFU settings

### PR DESCRIPTION
**Pull request by nooges**
## Description

Update Iris Rev 3 QMK-DFU settings with correct pinout

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by drashna** [comment]
Ugh oh..... looks like there is a bug in the ENCODER support for Split Common. 

Aside from that, looks good.

**Comment by drashna** [comment]
@nstickney You keymap is erroring out here, because of a change to the rgb light hue range. 

You don't need to change anything, but wanted to give you a heads up.

**Comment by nstickney** [comment]
Thanks for the heads-up!

File: keyboards/keebio/iris/rules.mk [hunk 1]
```diff
@@ -8,7 +8,7 @@ F_USB = $(F_CPU)
 #     different sizes, comment this out, and the correct address will be loaded
 #     automatically (+60). See bootloader.mk for all options.
 ifeq ($(strip $(KEYBOARD)), iris/rev3)
```
Comment (vostok92) [inline keyboards/keebio/iris/rules.mk]: This line must be:
> `ifeq ($(strip $(KEYBOARD)), keebio/iris/rev3)`

because folder structure was changed.

Comment (nooges) [reply keyboards/keebio/iris/rules.mk]: Thanks for pointing this out, I've submitted a fix here that makes the revision detection independent of folder structure: https://github.com/qmk/qmk_firmware/pull/6226

**Pull request merged**